modules/API/external/external_api.js

```diff
@@ -207,11 +206,32 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
      * @private
      */
     _createIFrame(height, width) {
+        // Compute valid values for height and width. If a number is specified
+        // it's treated as pixel units and our minimum constraints are applied.
+        // If the value is expressed in em, pt or percentage, it's used as is.
+        // Also protect ourselves from undefined, because
+        // Math.max(undefined, 100) === NaN, obviously.
+        const re = /([0-9]*\.?[0-9]+)(em|pt|%)$/;
```

Comment:
100pt, 100em, 100%. If it's none of those, it's either 100px or just 100 (which is the same to us) or invalid, which we will treat as the minimum.